kern/vmx.c

```diff
@@ -70,12 +70,12 @@ static DECLARE_BITMAP(vmx_vpid_bitmap, VMX_NR_VPIDS);
 static DEFINE_SPINLOCK(vmx_vpid_lock);
 
 typedef struct posted_interrupt_desc {
-    u32 vectors[8]; /* posted interrupt vectors */
-    u32 extra[8]; /* outstanding notification indicator and extra space the VMM can use */
+	u32 vectors[8];	/* posted interrupt vectors */
+	u32 extra[8];	/* outstanding notification indicator and extra space the VMM can use */
 } __aligned(64) posted_interrupt_desc;
 
 static unsigned long *msr_bitmap;
-//static struct vmx_vcpu *vcpus_hash[NR_CPUS];
+static struct vmx_vcpu *vcpus_hash[NR_CPUS];
```

Comment:
What is the use of this struct?

Reply:
That was originally present in Dune... I commented it out by mistake when implementing IPIs